### DISCUSSION TITLE
Autocomplete needs to add a defaultHoveredFirstOption parameter

### DISCUSSION THREAD

**Discussion by 104gogo** [mantinedev/mantine#1634] [Feature requests]
When we developed the search function, we used the Autocomplete component. When the API returned data, the results were displayed in the drop-down box of the Autocomplete component.

The problem we're having now is that the Autocomplete component selects the first item by default, and then the user presses enter to go to the details page of the first search result. But this is not the expected behavior, we want the user to enter the search results page after pressing Enter, and only enter the details page when the user clicks on an item.

Hope to add a parameter, so can not select the first item by default. thanks！

|  Name   | Type  | Description | Value
|  ----  | ----  | ---- | ---- |
| defaultHoveredFirstOption  | boolean | Set whether to hover the first item by default | true

**Comment by rtivital** [comment]
Hi, thanks for bringing that up! We'll change the logic of hovered items in version 5.0 to allow your use case.
